### PR TITLE
Support for .NET 8

### PR DESCRIPTION
**Pull request by tbragaf**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <Project>
   <PropertyGroup>
-    <DotnetVersion>net6.0</DotnetVersion>
+    <TargetDotnetVersions>net6.0;net8.0</TargetDotnetVersions>
+    <TargetDotnetVersionsWindows>net6.0-windows;net8.0-windows</TargetDotnetVersionsWindows>
```
Comment (joaompneves) [inline Directory.Build.props:4]: i dont find this in google

Comment (joaompneves) [inline Directory.Build.props:4]: why this difference? you should split versions and platforms and concat then both

Comment (tbragaf) [reply Directory.Build.props:4]: trying to do that as we speak :)
having some issues, but i'll keep you posted

File: NuGet.config [hunk 2]
```diff
@@ -12,10 +12,14 @@
 	</packageRestore>
 	
 	<packageSources>
+		<add key="Nuget2" value="/Users/tfs/Downloads" />
```
Comment (joaompneves) [inline NuGet.config:15]: !

**Review by joaompneves — APPROVED** [review]

File: Directory.Build.props [hunk 3]
```diff
@@ -1,10 +1,11 @@
 <Project>
   <PropertyGroup>
-    <DotnetVersion>net6.0</DotnetVersion>
+    <DefaultTargetDotnetVersion>net8.0</DefaultTargetDotnetVersion>
+    <TargetDotnetVersions>net6.0;$(DefaultTargetDotnetVersion)</TargetDotnetVersions>
     <Platforms>x64;ARM64</Platforms>
     <AssemblyVersion>2.0.0.0</AssemblyVersion>
     <FileVersion>2.0.0.0</FileVersion>
-    <Version>2.120.2</Version>
+    <Version>2.120.3</Version>
```
Comment (alvesmiguel1) [inline Directory.Build.props:8]: Just noticed it now, but maybe a relevant change worth it to bump the minor version

**Review by alvesmiguel1 — APPROVED** [review]

File: Directory.Build.props (hunk 3 above)
Comment (joaompneves) [reply Directory.Build.props:8]: well the minor is reserved for the cef version, well we used rev .100 for avalonia 11

**Pull request merged**